nvme_gw.config

```diff
@@ -25,6 +29,8 @@ client_cert = ./client.crt
 
 [spdk]
 
+spdk_path = /path/to/spdk
+spdk_tgt = spdk/build/bin/nvmf_tgt
```

Comment:
How do you feel about `parent_dir` and `tgt_path`?

Reply:
I don't think `parent_dir` is a good idea. Note, currently `spdk_path` is used for two things:
1) when looking for `nvmf_tgt` binary (in combination with `spdk_tgt` param)
2) when looking for python `spdk.scripts.rpc` module (we add `spdk_path` to `sys.path` before calling the module import)
`parent_dir` name is good for (1) but not for (2). And actually I don't like much that we do (1), I think we could just specify the full path in `spdk_tgt`.
And actually I am not sure we will need (2) in the future. It will depend on how we will ship ceph-nvmeof (and spdk). Ideally it should not be a configuration param.

So you probably leave `spdk_path` as is, because I expect it will change in the future anyway, unless someone propose a better name. `tgt_path` is ok for me (not ideal, but I can't think of anything better).